mlkem/ntt.c

```diff
@@ -53,7 +53,8 @@ __contract__(
 {
   /* `bound` is a ghost variable only needed in the CBMC specification */
   ((void)bound);
```

Comment:
How is that compliant? Isn't this technically a statement?

Reply:
resolved.